src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -343,8 +331,9 @@ object HttpGoogleServicesDAO extends GoogleServicesDAO with FireCloudRequestBuil
           // user is known to Google. Extract the user's email and SID from the response, for logging
           import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol.impOAuthUser
           import spray.json._
-          val oauthUser:Try[OAuthUser] = Try(userResponse.entity.asString.parseJson.convertTo[OAuthUser])
+          val oauthUser = Try(Unmarshaller.stringUnmarshaller.map(_.parseJson.convertTo[OAuthUser]))
           val userStr = (oauthUser getOrElse userResponse.entity).toString
+          userResponse.discardEntityBytes()
```

Comment:
here - and a few places below - it's important to `discardEntityBytes` if we don't consume the response entity in any other way. Else, akka stalls holding on to that entity.